### ISSUE TITLE
Fixed Issue when using python 2.7

### ISSUE THREAD

**Issue by amlarraz** [surmenok/keras_lr_finder#9]
Hi! Congrats for your job! I've found an Issue when use python 2.7 (I dont know if the python version is the problem), in file [lr_finder.py](keras_lr_finder/keras_lr_finder/lr_finder.py) in line 42 I've replaced:

` self.lr_mult = (end_lr / start_lr) ** (1 / num_batches)`

with

`self.lr_mult = (float(end_lr) / float(start_lr)) ** (float(1) / float(num_batches))`

because if I use the first line, the result was 1 all the time and the learning rate did not update at all.

**Comment by surmenok** [comment]
Good catch! Would you mind to submit a pull request?

**Comment by amlarraz** [comment]
I just did it!
